release/release_gold.sh

```diff
@@ -97,6 +98,252 @@ function main {
 	#lc_time_run upload_to_docker_hub
 
 	lc_time_run add_patcher_project_version
+
+	lc_time_run reference_new_releases
+}
+
+function reference_new_releases {
+	if [[ "${_PRODUCT_VERSION}" != *q* ]]
+	then
+		lc_log INFO "Skipping the update to the references in the liferay-jenkins-ee repository."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+
+	prepare_branch_to_commit_from_master "${_PROJECTS_DIR}/liferay-jenkins-ee/commands" "new_releases_branch"
+
+	if [ "${?}" -ne 0 ]
+	then
+		lc_log ERROR "Unable to prepare the next release references branch."
+
+		return "${LIFERAY_COMMON_EXIT_CODE_BAD}"
+	fi
+
+	local previous_quarterly_release_branch_name="$(grep "extraapps.app.server.versions" "build.properties" | tail -1 | cut -d '[' -f 2 | cut -d ']' -f 1)"
+
+	if [[ "$(echo "${_PRODUCT_VERSION}" | cut -d '.' -f 3)" -eq 0 ]] && ! grep -q "${LIFERAY_RELEASE_GIT_REF}" "build.properties"
+	then
+		sed \
+			-i "s/^		${previous_quarterly_release_branch_name}/		${previous_quarterly_release_branch_name},\\\\/" \
+			"build.properties"
+
+		sed \
+			-i "/${previous_quarterly_release_branch_name},/a\		${LIFERAY_RELEASE_GIT_REF}" \
+			"build.properties"
+
+		sed \
+			-i "/extraapps.app.server.versions\[${previous_quarterly_release_branch_name}\]=/a\	extraapps.app.server.versions[${LIFERAY_RELEASE_GIT_REF}]=tomcat8" \
+			"build.properties"
+
+		sed \
+			-i "/extraapps.database.versions\[${previous_quarterly_release_branch_name}\]=/a\	extraapps.database.versions[${LIFERAY_RELEASE_GIT_REF}]=mysql57" \
+			"build.properties"
+
+		sed \
+			-i "/faces.alloy.branch.name\[${previous_quarterly_release_branch_name}\]=/a\	faces.alloy.branch.name[${LIFERAY_RELEASE_GIT_REF}]=master" \
+			"build.properties"
+
+		sed \
+			-i "/faces.bridge.impl.branch.name\[${previous_quarterly_release_branch_name}\]=/a\	faces.bridge.impl.branch.name[${LIFERAY_RELEASE_GIT_REF}]=4.x" \
+			"build.properties"
+
+		sed \
+			-i "/faces.portal.branch.name\[${previous_quarterly_release_branch_name}\]=/a\	faces.portal.branch.name[${LIFERAY_RELEASE_GIT_REF}]=4.x" \
+			"build.properties"
+
+		sed \
+			-i "/faces.showcase.branch.name\[${previous_quarterly_release_branch_name}\]=/a\	faces.showcase.branch.name[${LIFERAY_RELEASE_GIT_REF}]=3.x" \
+			"build.properties"
```

Comment:
@kiwm, isn't this code going beyond the expected scope? Please see an example of the expected scope here: https://github.com/pyoo47/liferay-jenkins-ee/pull/3140/commits/d169d8d0b87154ce918e54d2dd570a6d19080f01.

Let's chat tomorrow about this pull. Please reach out me offline.